### PR TITLE
[[ ExtensionDocs ]] Don't manually update docs on extension install

### PR DESCRIPTION
**Pull request by livecodeali**
This patch adds a subscription to ideExtensionsChanged to the ide
library, which when received causes the built dictionary data to
be regenerated.

### CODE REVIEW

File: Toolset/libraries/revidelibrary.8.livecodescript [hunk 1]
```diff
@@ -1513,8 +1518,14 @@ on revIDEInitialiseIDELibrary
    
    ## Initialise property library
    revIDEPropertyLibraryInitialise
+   
+   ideSubscribe "ideExtensionsChanged"
 end revIDEInitialiseIDELibrary
 
+on ideExtensionsChanged
+   revIDERegenerateBuiltDictionaryData
```
Comment (montegoulding) [inline Toolset/libraries/revidelibrary.8.livecodescript:1526]: @livecodeali would it be a good idea to add in a delay and cancel here so if multiple extensions are loaded the docs are regenerated only once?

Comment (livecodeali) [reply Toolset/libraries/revidelibrary.8.livecodescript:1526]: I think it's maybe not much of an issue, as the time to reassemble the main `.js` is pretty tiny, especially compared to the time taken to generate the json from an `api.lcdoc`

Comment (montegoulding) [reply Toolset/libraries/revidelibrary.8.livecodescript:1526]: OK, cool

**Comment by montegoulding** [comment]
@livecode-vulcan review ok 322f8ab

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok 322f8ab9de8477e0d6021fb725764d9794e92523

**Pull request merged**